Web.md

```diff
@@ -14,6 +14,12 @@ than if the module was JavaScript.
 More concretely, the following is a list of points of contact between WebAssembly
 and the rest of the Web platform that have been considered:
 
+## The `Wasm` object
+
+WebAssembly is exposed to the Web through a `Wasm` object, whose API is designed
```

Comment:
Would you rather I simply rephrase as "Besides [modules](LINK), WebAssembly is exposed to the Web through..." ?

That should be in its own section IMO, and currently isn't even A Thing.